hypothesis-ruby/src/lib.rs

```diff
@@ -50,6 +50,7 @@ wrappable_struct!(
 
 class!(HypothesisCoreDataSource);
 
+#[rustfmt::skip]
```

Comment:
Thank you - I was manually editing these back to be able to compile after running `cargo fmt` because the formatting doesn't work for these macros.

Reply:
You're welcome! :) It would be nice to have more fine-grained formatting inside methods (e.g., so method bodies are formatted), though. But not sure what would be the best approach for this without adding much noise by using such attributes

Reply:
And, alas, [there are plenty of open issues about `rust-fmt` breaking macros](https://github.com/rust-lang/rustfmt/issues?q=is%3Aissue+is%3Aopen+macro) 😕

Looks like this approach is the best available for now.